templates/satellite-answers.yaml

```diff
@@ -24,18 +24,18 @@ foreman:
   custom_repo: true
   configure_epel_repo: false
   configure_scl_repo: false
-  server_ssl_cert: /etc/pki/katello/certs/katello-apache.crt
-  server_ssl_key: /etc/pki/katello/private/katello-apache.key
-  server_ssl_ca: /etc/pki/katello/certs/katello-default-ca.crt
-  server_ssl_chain: /etc/pki/katello/certs/katello-server-ca.crt
+  server_ssl_cert: "/etc/pki/katello/certs/katello-apache.crt"
```

Comment:
The quotation marks *should* be unnecessary. But also not a problem.

Reply:
I know about that. I just wanted to keep a track with the Satellite's original answer file.